tests/ios/xcconfig/WithDefinesObjc/WithDefines.m

```diff
@@ -0,0 +1,30 @@
+@import Foundation;
+
+@interface WithDefinesObjc : NSObject
+@end
+
+@implementation WithDefinesObjc
+
+#if !MACRO_A
+ thisShouldNotBeCompiled
```

Comment:
Would `#error "MACRO_A should be set via merged xcconfig"` more clearly indicate compilation is expected to fail when evaluating to true?